tests/FSharp.Compiler.Service.Tests/SurfaceArea.netstandard.fs

```diff
@@ -3990,27 +3990,153 @@ FSharp.Compiler.EditorServices.XmlDocable: Int32 line
 FSharp.Compiler.EditorServices.XmlDocable: Microsoft.FSharp.Collections.FSharpList`1[System.String] get_paramNames()
 FSharp.Compiler.EditorServices.XmlDocable: Microsoft.FSharp.Collections.FSharpList`1[System.String] paramNames
 FSharp.Compiler.EditorServices.XmlDocable: System.String ToString()
+FSharp.Compiler.IO.ByteArrayMemory
```

Comment:
Are these now public? They should be internal only.

Reply:
Since we expose ByteMemory type now via the IFileSystem interface, I was thinking to make ByteArrayMemory (and maybe other ones) public too, as default implementations, so FCS consumers can use it in their IFileSystem implementations. WDYT?

Reply:
`ByteMemory` and friends should really be internal if possible.

Reply:
@vzarytovskii and I talked a bit, we will expose these types but will rename them with the `FSharp` prefix and will be given experimental attributes as we might make them internal again and/or change them as we continue to iterate on this.